### PR TITLE
Enable Italian Translation

### PR DESCRIPTION
**Pull request by dngray**
Changes proposed in this PR:

- 

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### CODE REVIEW

**Comment by jonaharagon** [comment]
Netlify's deploy preview doesn't generate translated sites, the *branch* deploy does, and is available here:

https://i18n-it.staging.privacyguides.dev/it/

**Comment by mfwmyfacewhen** [comment]
It seems like the highlight isn't working in the preview for some reason.

**Comment by blacklight447** [comment]
Graph on the dns page is broken

File: config/mkdocs.es.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ copyright: |
...
   Contenido bajo licencia <a href="/license"><strong>CC BY-ND 4.0</strong></a>.
-edit_uri: edit/main/docs/
+edit_uri: edit_uri: edit/main/i18n/es/
```
Comment (jonaharagon) [inline config/mkdocs.es.yml:35]:
```suggestion
edit_uri: edit/main/i18n/es/
```

oops

**Review by mfwmyfacewhen — APPROVED** [review]

**Review by blacklight447 — APPROVED** [review]

**Pull request merged**